src/Microsoft.AspNetCore.Identity/IIdentityBuilderExtensions.cs

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Identity
+{
+    /// <summary>
+    /// Helper functions for configuring identity services.
+    /// </summary>
+    public static class IIdentityBuilderExtensions
```

Comment:
Naming: Don't include the "I" for interfaces in the name of the extension method sponsor class, i.e. this should be just `IdentityBuilderExtensions`.